### DISCUSSION TITLE
编译成rn时报缺少项目配置

### DISCUSSION THREAD

**Discussion by hjw6685** [NervJS/taro#14299] [Q & A]
我拉取了一个最新的react-native模板后，安装完依赖，然后编译成rn会报缺少项目基本配置，有没有大佬有具体流程分享分享

**Comment by Barrierml** [comment]
缺少项目基本配置 只是 tansform 的配置，并不会影响项目的主流程运行，确定 upgradePeerdeps 命令与 pod install 运行完毕后，直接 yarn ios 或 android 项目就可以正常运行了

<img width="1433" alt="image" src="https://github.com/NervJS/taro/assets/40788017/d0dfcd7b-d46d-428b-be18-d669edd221b3">

可以参考官方的运行文章
https://docs.taro.zone/blog/2021-10-14-Taro-React-Native-update/#react-native-%E5%BC%80%E5%8F%91%E6%A8%A1%E6%9D%BF

**Comment by hjw6685** [comment]
安卓电脑也要执行pod install吗，这个不是只有ios的时候执行，执行upgradePeerdeps&nbsp;完还是一样的配置错误，tansform 配置应该配什么才能让报这个消失




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "NervJS/taro"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年8月9日(星期三) 中午11:28
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [NervJS/taro] 编译成rn时报缺少项目配置 (Discussion #14299)





 
缺少项目基本配置 只是 tansform 的配置，并不会影响项目的主流程运行，确定 upgradePeerdeps 命令与 pod install 运行完毕后，直接 yarn ios 或 android 项目就可以正常运行了
  
可以参考官方的运行文章
 https://docs.taro.zone/blog/2021-10-14-Taro-React-Native-update/#react-native-%E5%BC%80%E5%8F%91%E6%A8%A1%E6%9D%BF
